History: show entries in the order in which they were read

pdewacht opened the pull request:
With these changes, the history view will show entries in the order in which they were read.

This is implemented by adding a changed_at field in the Entries table which tracks when the entry's metadata was changed.

I did this because I often had a hard time finding an entry again after it was marked as read (accidentally or not), but this would also partially address issue #453 and perhaps #298 and #417.

fguillot commented on ui/history_entries.go:
```diff
@@ -25,7 +25,7 @@ func (h *handler) showHistoryPage(w http.ResponseWriter, r *http.Request) {
 	builder.WithStatus(model.EntryStatusRead)
-	builder.WithOrder(model.DefaultSortingOrder)
+	builder.WithOrder("changed_at")
 	builder.WithDirection(user.EntryDirection)
```
What about the direction? The default value is `ASC`.

pdewacht replied:
Yeah, I didn't notice that. Fixed. As you might guess, I use "recent entries first". :)

immanuelfodor commented:
Thanks @pdewacht for this PR, could you please update it according to the code review comment (using `DESC` order) to make it mergeable? :) I recently run into the same issue that I "lost" an accidentally read article and had hard time to find it in the history. I'd expect it to be on the top of the history page as most recent read item. It would be great to have this PR merged to solve our common issue.

pdewacht commented:
Sorry for disappearing. I've rebased this PR onto master and fixed the direction issue.

I've been using this change on top of 2.0.18 for the past few months and I've had no problems with it.

clemera commented:
It would be great to also have this feature for querying entries from the REST API.

pdewacht commented:
I think you can use `order=changed_at` to get this with the API.

clemera commented:
@pdewacht Nice thanks!
